pkgs/tools/audio/tts/tts.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/requirements.txt b/requirements.txt
+index e3871874..3e3a881f 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -21,7 +21,6 @@ umap-learn==0.5.1
+ pandas
+ # deps for training
+ matplotlib
+-tensorboardX
```

Comment:
We don't carry patches for minimal changes like this. Use `sed` in `postPatch` if you must.

Reply:
I've checked tts with git grep but it does not use tensorboardX.

Reply:
https://github.com/coqui-ai/TTS/issues/1433

Reply:
```patch
diff --git a/pkgs/tools/audio/tts/default.nix b/pkgs/tools/audio/tts/default.nix
index a7b0212a7b0..a15a29f9d57 100644
--- a/pkgs/tools/audio/tts/default.nix
+++ b/pkgs/tools/audio/tts/default.nix
@@ -57,6 +57,7 @@ python.pkgs.buildPythonApplication rec {
         ''-e 's/${package}.*[<>=]+.*/${package}/g' \''
       ) relaxedConstraints)}
     requirements.txt
+    sed -i '/tensorboardX/d' requirements.txt
   '';
 
   nativeBuildInputs = with python.pkgs; [
@@ -84,7 +85,6 @@ python.pkgs.buildPythonApplication rec {
     pyworld
     scipy
     soundfile
-    tensorboardx
     tensorflow
     torchaudio-bin
     tqdm

```

Reply:
Can you please fixup that commit and squash it into the previous one?